src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.application.name=ms-ingress-spring
```

Comment:
use yaml format (application.yaml instead of application.properties)
spring:
  application:
    name: ms-ingress-spring

Reply:
I used application.yml instead of application.properties. Is everything okay now?